code/__HELPERS/spatial_info.dm

```diff
@@ -110,11 +110,9 @@
 	//by default, for(var/atom/name in view()) iterates through both the /obj linked list then the /mob linked list of each turf
 	//but because what we want are only a tiny proportion of all movables, most of the things in the /obj contents list are not what we're looking for
 	//while every mob can hear. for this case view() has an optimization to only look through 1 of these lists if it can (eg youre only looking for mobs)
-	//so by representing every hearing contents on a turf with a single /mob/oranges_ear containing references to all of them, we are:
-	//1. making view() only go through the smallest of the two linked lists per turf, which contains the type we're looking for at the end
-	//2. typechecking all mobs in the output to only actually return mobs of type /mob/oranges_ear
-	//on a whole this can outperform iterating through all movables in view() by ~2x especially when hearables are a tiny percentage of movables in view
-	for(var/mob/oranges_ear/ear in view(view_radius, center_turf))
+	//so by representing every hearing contents on a turf with a single /mob/oranges_ear containing references to all of them, we are
+	//able to use hearers() over view(), which is a huge speed increase.
+	for(var/mob/oranges_ear/ear in hearers(view_radius, center_turf))
```

Comment:
You should remove setting the center turfs luminosity too

Reply:
I thought I did, forgot to push